hepstats/hypotests/fitutils/sampling.py

```diff
@@ -0,0 +1,62 @@
+from .api_check import is_valid_pdf
+
+"""
+Module providing basic sampling methods.
+"""
+
+
+def base_sampler(models, nevents, floating_params=None):
+    """
+    Creates a samplers from models.
```

Comment:
a sampler

Reply:
Oops, should rather remove the "a" since you return several samplers.

There are some fixes to be made to the Args section ... (words missing, optional attribute).